.github/workflows/trivy.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: build
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '15 0 * * *'
+
+jobs:
+  scan:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+    name: Scan with Trivy
+    strategy:
+      matrix:
+        image:
+          - main
+          - roxctl
+          - central-db
+    runs-on: ubuntu-18.04
+    steps:
+          - name: Run Trivy vulnerability scanner in repo mode
+            uses: aquasecurity/trivy-action@master
+            with:
+              image-ref: 'quay.io/stackrox-io/${{ matrix.image }}:latest'
```

Comment:
It looks good in general, but ca we be sure that the `latest` will always correspond to the image built in the current PR?